gammapy/maps/base.py

```diff
@@ -499,6 +499,32 @@ def upsample(self, factor, order=0, preserve_counts=True):
         """
         pass
 
+    def slice_by_idx(self, slices, copy=False):
+        """Slice sub map from map object.
+
```

Comment:
Add reference to the high-level docs section with the examples?

Reply:
Done